### DISCUSSION TITLE
Ability to change live timing layout by adding 2 more columns on full width

### DISCUSSION THREAD

**Discussion by evertconrad** [f1multiviewer/issue-tracker#87] [Feature requests] [answered]
Basically what i would like to have implemented in a future update is the option to switch between a 1/2/4 columns layout in live timing window.

Especially when u keep the window on full width at the bottom of the screen the live timing will have plenty of unused space that could be used with 2 more columns of data
This a normal layout in full width: https://cdn.discordapp.com/attachments/984792219573694465/992762190341881906/normal-layout.png
This is what it should look like with 2 more columns: https://cdn.discordapp.com/attachments/984792219573694465/992762211028185149/alternative-layout.jpg

**Comment by JJWatMyself** [comment]
Agreed.

**Comment by Walker1390** [comment]
In addition could we get the ability to have the timings on one side and races control messages on the other, splitting them down the middle vertically instead of horizontally?

**Comment by f1multiviewer** [comment, marked as answer]
I've added a 4-column version in the next release! A 3-column version didn't work well because of layout issues (due to 20/3 not being a integer division)
